jobs/disabled-aws-regions.Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+import org.yaml.snakeyaml.Yaml;
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+}
+
+properties([
+    pipelineTriggers([
+        githubPush(),
+        // once in every two hours slot between 9 AM and 5 PM every weekday
+        pollSCM('H H(9-16)/2 * * 1-5')
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '100',
+        artifactNumToKeepStr: '100'
+    )),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+cosaPod(){
+    withCredentials([file(variable: 'AWS_CONFIG_FILE', credentialsId: 'aws-kola-tests-config')]) {
+        def disabled_regions = shwrapCapture("ore aws list-regions --disabled")
```

Comment:
we can end the credentials closure here (terminate the `{` with a closing `}`) if we define `def disabled_regions = ""` before the `withCredentials`.